benchmarks/perl-5.21.7/Dockerfile

```diff
@@ -0,0 +1,19 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+FROM gcr.io/oss-fuzz-base/base-builder
+COPY build.sh perl_fuzz.cc $SRC/
+ADD seeds /opt/seeds
```

Comment:
missing new line

Reply:
Don't really know where you wanted it, on line 19? I added one there and 18 since we usually have one after from.

Reply:
i mean end of file new-line char missing (after line 19), showed that red symbol in code-review.